src/monodroid/jni/util.cc

```diff
@@ -173,6 +173,24 @@ Util::monodroid_store_package_name (const char *name)
 	log_info (LOG_DEFAULT, "Generated hash 0x%s for package name %s", package_property_suffix, name);
 }
 
+#if defined (NET6)
+MonoAssembly*
+Util::monodroid_load_assembly (MonoAssemblyLoadContextGCHandle alc_handle, const char *basename)
```

Comment:
Why does this member function exist when it doesn't appear to be used from anywhere? Dead code?

Reply:
No, I planned to start using it in another PR, one would review all the usage of AppDomains across the code. I don't want to do it in this PR to keep it more focused. However, since this method will be called from few places, I'll add the calls to this PR.